src/components/AddLiquidityModal.js

```diff
@@ -0,0 +1,133 @@
+import React, { useContext, useState } from "react";
+
+import { calcNearAddLiquidity, convertToE24Base5Dec, incAllowance, addLiquidity } from "../services/near-nep21-util";
+
+import Modal from 'react-bootstrap/Modal';
+import Button from 'react-bootstrap/Button';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+
+import { InputsContext } from "../contexts/InputsContext";
+
+import { BsCaretDownFill } from "react-icons/bs";
+
+import styled from "@emotion/styled";
+const InputBox = styled("div")`
+  color: ${props => props.theme.body};
+  border: 1px solid #dee2e6;
+  border-radius: 20px;
+  .form-control:focus {
+    color: #212543;
+  }
+`;
```

Comment:
Let's keep style definitions in `CSS` / `LESS` / `SASS` files.

Reply:
Typically projects using emotion or styled-components have localized style definitions per component, but I will create a separate issue for this anyways because some of the style definitions are messy and need refactoring

Reply:
OK, please don't forget to create an issue.